include/pops/network.hpp

```diff
@@ -144,43 +171,42 @@ class Network
      *
      * ```
      * Network network(...);
-     * std::string node_filename = "nodes.txt";
-     * std::string segment_filename = "segments.txt"
-     * std::ifstream node_stream{node_filename};
-     * std::ifstream segment_stream{segment_filename};
-     * network.load(node_stream, segment_stream);
+     * std::string filename = "network.txt"
+     * std::ifstream stream{filename};
+     * network.load(stream);
      * ```
      *
-     * The input network is a list of nodes and their coordinates and a list of edges
+     * The input network a list of edges which are pairs of nodes
```

Comment:
```suggestion
     * The input network is a list of edges which are pairs of nodes
```
Should it rather be "list of segments which are pairs of nodes and the corresponding edges"?
Or "where each segments consists of a pair of nodes and a corresponding edge".